app/views/search/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= search_form_for @q do |f| %>
   <div class="form-group">
     <%= f.label t("exercise_search") %>
-    <input class="form-control" id="q_language_name_or_title_or_description_or_locale_cont" name="q[language_name_or_title_or_description_or_locale_cont]" type="search">
+    <input class="form-control" id="q_language_name_tags_name_or_title_or_description_or_locale_cont" name="q[language_name_or_tags_name_or_title_or_description_or_locale_cont]" type="search">
```

Comment:
Another issue I noticed: this is actually searching by any of the fields, but not by a combination of them. That means, if i search "haskell basic", it will find exercies that match "haskell" or "basic" but not both - e.g. it will not find an exercise in the Haskell language with tag "basic".